### PR TITLE
Update bundled minishift.yaml to correspond to jboss-stack's yaml file

### PR DESCRIPTION
**Pull request by odockal**
Signed-off-by: Ondrej Dockal <odockal@redhat.com>

# Pull Request Checklist
## General
* Is this a blocking issue or new feature? If yes, QE needs to +1 this PR

## Code
* Are method-/class-/variable-names meaningful?
* Are methods concise, not too long?
* Are catch blocks catching precise Exceptions only (no catch all)?

## Testing
* Are there unit-tests?
* Are there integration tests (or at least a jira to tackle these)?
* Is the non-happy path working, too?
* Are other parts that use the same component still working fine?

## Function
* Does it work?

### CODE REVIEW

**Comment by odockal** [comment]
@robstryker I have run tests locally (checked via jboss-stack test class via parse funcion and I tried to validate the yaml online and some validators show errors (https://codebeautify.org/yaml-validator) while some do not (http://www.yamllint.com/) . I believe, that depends on version of yaml version used for parsing. That led me to the jboss-stack implementation of YAML. there is used snakeyaml 1.13 version injected as maven dependency. That is version made back in 2013. This is pretty old. We might want to update this repo used for stacks (org.jboss.jdf), right?

**Comment by odockal** [comment]
Here are new unit tests for stack runtimes: https://dev-platform-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/jbosstools-openshift-Pull-Request/1721/testReport/org.jboss.tools.openshift.cdk.server.test.internal/CDKStackRuntimeTest/

File: test-framework/org.jboss.tools.cdk.reddeer/src/org/jboss/tools/cdk/reddeer/core/enums/CDKVersion.java [hunk 1]
```diff
@@ -29,7 +29,8 @@
 	MINISHIFT1140 ("Minishift", "1.14.0", CDKLabel.Server.MINISHIFT_SERVER_NAME, "minishift-1.14.0-" + CDKRuntimeOS.get().getRuntimeFullName() + "-amd64"),
 	MINISHIFT1151 ("Minishift", "1.15.1", CDKLabel.Server.MINISHIFT_SERVER_NAME, "minishift-1.15.1-" + CDKRuntimeOS.get().getRuntimeFullName() + "-amd64"),
 	MINISHIFT1161 ("Minishift", "1.16.1", CDKLabel.Server.MINISHIFT_SERVER_NAME, "minishift-1.16.1-" + CDKRuntimeOS.get().getRuntimeFullName() + "-amd64"),
-	MINISHIFT1170 ("Minishift", "1.17.0", CDKLabel.Server.MINISHIFT_SERVER_NAME, "minishift-1.17.0-" + CDKRuntimeOS.get().getRuntimeFullName() + "-amd64");
+	MINISHIFT1170 ("Minishift", "1.17.0", CDKLabel.Server.MINISHIFT_SERVER_NAME, "minishift-1.17.0-" + CDKRuntimeOS.get().getRuntimeFullName() + "-amd64"),
+	MINISHIFT1180 ("Minishift", "1.18.0", CDKLabel.Server.MINISHIFT_SERVER_NAME, "minishift-1.18.0-" + CDKRuntimeOS.get().getRuntimeFullName() + "-amd64");
```
Comment (robstryker) [inline test-framework/org.jboss.tools.cdk.reddeer/src/org/jboss/tools/cdk/reddeer/core/enums/CDKVersion.java]: Why not add 1.19 and 1.20 here?

Comment (odockal) [reply test-framework/org.jboss.tools.cdk.reddeer/src/org/jboss/tools/cdk/reddeer/core/enums/CDKVersion.java]: True that. corrected.

**Comment by odockal** [comment]
@robstryker I have rebased the branch to have all at place. Please merge when build is complete. Thanks!

**Comment by odockal** [comment]
@robstryker all good I think.

**Comment by jkopriva** [comment]
LGTM, merging :)

**Pull request merged**